stdlib/tkinter/__init__.pyi

```diff
@@ -667,12 +670,24 @@ class Wm:
         self, minNumer: None = None, minDenom: None = None, maxNumer: None = None, maxDenom: None = None
     ) -> tuple[int, int, int, int] | None: ...
     aspect = wm_aspect
-    @overload
-    def wm_attributes(self) -> tuple[Any, ...]: ...
+    if sys.version_info >= (3, 13):
+        @overload
+        def wm_attributes(self, *, return_python_dict: Literal[False] = False) -> tuple[Any, ...]: ...
+        @overload
+        def wm_attributes(self, *, return_python_dict: Literal[True]) -> dict[str, Any]: ...
```

Comment:
Instead of `Any`, we could use a `TypedDict` to represent what exactly is returned. You can either do it in this PR or leave a comment like this:

```suggestion
        def wm_attributes(self, *, return_python_dict: Literal[True]) -> dict[str, Any]: ...  # TODO: use TypedDict
```